ext/build_defaults.yaml

```diff
@@ -146,3 +146,4 @@ vanagon_project: TRUE
 apt_repo_name: 'PC1'
 yum_repo_name: 'PC1'
 build_tar: FALSE
+build_gem: TRUE
```

Comment:
@underscorgan I looked at puppet's build_defaults (and a little in packaging) and this looked like the only thing I needed to enable to get gems to ship. Does that sound right to you?

Reply:
@McdonaldSeanp that's the only thing I know of!

Reply:
Missing newline here. I dunno if it'll confuse any real-world YAML parsers, but I think it's technically out of spec :)

Reply:
@branan updated